### PR TITLE
Implement dome view and move nightly to dome

### PR DESCRIPTION
**Pull request by j-rivero**
The PR implements the nightly generation for Dome versions. It needed an auxiliary function to generate only a subset of nightlies.

Tested: [Build Status](https://build.osrfoundation.org/job/_dsl_ignition_collection/212/)

Ignition debbuilder tested here: [Build Status](https://build.osrfoundation.org/view/All/job/_dsl_ignition/1001/)

```
Added items:
    GeneratedJob{name='ign-fuel-tools5-debbuilder'}
    GeneratedJob{name='ign-gazebo4-debbuilder'}
    GeneratedJob{name='ign-gui4-debbuilder'}
    GeneratedJob{name='ign-launch3-debbuilder'}
    GeneratedJob{name='ign-msgs6-debbuilder'}
    GeneratedJob{name='ign-physics3-debbuilder'}
    GeneratedJob{name='ign-rendering4-debbuilder'}
    GeneratedJob{name='ign-sensors4-debbuilder'}
    GeneratedJob{name='ign-sdformat10-debbuilder'}
    GeneratedJob{name='ign-transport9-debbuilder'}
```

-release repositories needed for nightlies:
  * [x] ign-fuel-tools5
  * [x] ign-gazebo4
  * [x] ign-gui4
  * [x] ign-launch3
  * [x] ign-msgs6
  * [x] ign-physics3
  * [x] ign-rendering4
  * [x] ign-sensors4
  * [x] ign-transport9
  * [x] sdformat10

### CODE REVIEW

File: jenkins-scripts/dsl/ignition_collection.dsl [hunk 1]
```diff
@@ -189,7 +189,85 @@ ignition_collection_jobs =
...
+        'ignition_common-ci-ign-common3-windows7-amd64',
+        'ignition_common3-install-pkg-bionic-amd64',
+        'ignition_common3-install_bottle-homebrew-amd64',
+        // 'ignition_fuel-tools-ci-ign-fuel-tools5-bionic-amd64',
+        // 'ignition_fuel-tools-ci-ign-fuel-tools5-homebrew-amd64',
+        // 'ignition_fuel-tools-ci-ign-fuel-tools5-windows7-amd64',
+        // 'ignition_fuel-tools5-install-pkg-bionic-amd64',
+        // 'ignition_fuel-tools5-install_bottle-homebrew-amd64',
+        // 'ignition_gazebo-ci-ign-gazebo4-bionic-amd64',
+        // 'ignition_gazebo-ci-ign-gazebo4-homebrew-amd64',
```
Comment (scpeters) [inline jenkins-scripts/dsl/ignition_collection.dsl:218]: I think we can use the master branch CI jobs for now until the release branches are available

Comment (j-rivero) [reply jenkins-scripts/dsl/ignition_collection.dsl]: 7209c43, running collection again.

**Comment by j-rivero** [comment]
The nightly run displays problems with missing -release repositories (all these MARK_AS_UNSTABLE):

```
releasing ign-fuel-tools5 (as ignition-fuel-tools5) from branch ign-fuel-tools5 --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-gazebo4 (as ignition-gazebo4) from branch not_enabled_in_DSL --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-gui4 (as ignition-gui4) from branch not_enabled_in_DSL --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-launch3 (as ignition-launch3) from branch not_enabled_in_DSL --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-math6 (as ignition-math6) from branch not_enabled_in_DSL --ignition-repo
 - done
releasing ign-msgs6 (as ignition-msgs6) from branch ign-msgs6 --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-physics3 (as ignition-physics3) from branch ign-physics3 --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-rendering4 (as ignition-rendering4) from branch ign-rendering4 --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing ign-sensors4 (as ignition-sensors4) from branch ign-sensors4 --ignition-repo
MARK_AS_UNSTABLE
 - done
releasing sdformat1 (as sdformat1) from branch sdf10 
MARK_AS_UNSTABLE
 - done
releasing ign-transport9 (as ignition-transport9) from branch ign-transport9 --ignition-repo
MARK_AS_UNSTABLE
 - done
```
math6 should not be part of the nightlies AFAIK.

File: jenkins-scripts/dsl/ignition_collection.dsl [hunk 2]
```diff
@@ -189,7 +188,85 @@ ignition_collection_jobs =
...
         'sdformat-ci-sdformat9-windows7-amd64',
         'sdformat-install-sdformat9_pkg-bionic-amd64'
-  ]
+  ],
+  'dome' : [
+        'ign_gazebo-master-win',
+        'ign_gui-maser-win',
+        'ign_physics-master-win',
+        'ign_rendering-master-win',
+        'ign_sensors-master-win',
```
Comment (scpeters) [inline jenkins-scripts/dsl/ignition_collection.dsl:197]: I think all these `ign_` branches use the format `ign_*-ci-win` for the master branch, not `*-master-win`

Comment (j-rivero) [reply jenkins-scripts/dsl/ignition_collection.dsl]: ouch thanks! d56c066

Comment (scpeters) [reply jenkins-scripts/dsl/ignition_collection.dsl]: one extra change in 7ad2c37 and now it looks good to me

**Pull request merged**